swap/src/bitcoin/lock.rs

```diff
@@ -200,9 +200,10 @@ mod tests {
     #[tokio::test]
     async fn bob_can_fund_without_a_change_output() {
         let (A, B) = alice_and_bob();
-        let fees = 610;
+        let fees = 300;
```

Comment:
Why did this number change?

Reply:
I think something must have changed in the upgrade to `bdk` ? The previous fee value resulted in a change output being included.

Reply:
I think it's probably 

> Fixed esplora fee estimation. 

https://github.com/bitcoindevkit/bdk/compare/v0.12.0...v0.16.1#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR24

Reply:
Maybe @LLFourn can elucidate 😅 

https://github.com/bitcoindevkit/bdk/commit/732166fcb689ad666eab02a4bb28c33018f7b819